src/view/edit-point-view.js

```diff
@@ -1,7 +1,32 @@
 import {createElement} from '../render.js';
 
-const createEditPointTemplate = () => (
-  `<li class="trip-events__item">
+const getListOffers = (offersArray) => {
+  let stringOffers = '';
+  let oldOffers = '';
+  if(offersArray !== null) {
+    for (const offer of offersArray) {
+      const eventOfferTitle = (offer.title !== null) ? offer.title : '';
+      const eventOfferPrice = (offer.price !== null) ? offer.price : '';
+      const bodyOffersItem = `
+      <div class="event__offer-selector">
+        <input class="event__offer-checkbox  visually-hidden" id="event-offer-luggage-1" type="checkbox" name="event-offer-luggage" checked="">
+        <label class="event__offer-label" for="event-offer-luggage-1">
+          <span class="event__offer-title cye-lm-tag">${eventOfferTitle}</span>+€&nbsp;<span class="event__offer-price cye-lm-tag">${eventOfferPrice}</span>
+        </label>
+      </div>`;
+      stringOffers = `${oldOffers}${bodyOffersItem}`;
+      oldOffers = stringOffers;
+    }
+    return stringOffers;
+  }
+  return stringOffers;
+};
+
+const createEditPointTemplate = (point) => {
+  const {description, name, offers, pictureDescription, picturesSrc, type, price, date} = point;
+  const listOffers = getListOffers(offers);
+
+  return (`<li class="trip-events__item">
```

Comment:
```suggestion
  return (
    `<li class="trip-events__item">
```

и дальше форматирование проверь.